source/docs/tuner/controlling-devices.rst

```diff
@@ -0,0 +1,31 @@
+Controlling Devices
+===================
+
+Motors can be controlled without any robot program running through Tuner X. When combined with :ref:`docs/tuner/plotting:plotting`, it can be an excellent tool for calculating closed loop gains or isolating mechanical issues.
```

Comment:
The first sentence is worded a bit weirdly, maybe something more like
"Tuner X can be used to directly control devices without a robot program."?